schemas/app-interface/app-interface-settings-1.yml

```diff
@@ -210,6 +210,22 @@ properties:
     - defaultIssueType
     - defaultPriority
     - defaultReopenState
+  customMessages:
+    type: array
+    description: custom messages to be used in integrations
+    items:
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+          description: unique id of the message
+        content:
+          type: string
+          description: content of the message
```

Comment:
Is that a Jinja templated string? I think that would be cool to be able to apply context info in messages.

Reply:
I guess this depends on the id being defined and how it's used in the code of course